content/applications/finance/accounting/payments/batch_sdd.rst

```diff
@@ -1,130 +1,174 @@
+   - The :guilabel:`SDD Scheme` depends on the type of customer: Select :guilabel:`CORE` for B2C
+     customers and :guilabel:`B2B` for B2B customers.
+   - |sdd| mandates are created automatically for :ref:`online payments made with SDD
+     <accounting/batch_sdd/online-payments>`.
+
+Closing or revoking a mandate
+-----------------------------
+
+|sdd| mandates are closed automatically after their :guilabel:`End Date`. If this field is
+left empty, the mandate remains active until it is closed or revoked. To close or revoke a mandate,
+go to :menuselection:`Accounting --> Customers --> Direct Debit Mandates`, select the relevant
+mandate, and click :guilabel:`Close` or :guilabel:`Revoke`.
+
+**Closing** a mandate updates the mandate's end day to the current day. Invoices issued after the
+present day will not be processed with an |sdd| payment. **Revoking** a mandate disables the
+mandate immediately. No |sdd| payment can be registered anymore, regardless of the invoice's
+date. However, payments that have already been registered are still included in the next :ref:`SDD
+XML file <accounting/batch_sdd/XML>`.
 
-.. todo::
-   Add link to future **Payment Provider** documentation.
+.. warning::
+   Closed or revoked mandates cannot be reactivated.
 
-Close or revoke a mandate
--------------------------
+.. _accounting/batch_sdd/online-payments:
 
-Direct debit mandates are closed automatically after their :guilabel:`End Date`. If this field is
-left blank, the mandate keeps being **Active** until it is **Closed** or **Revoked**.
+Online payments with |sdd|
+==========================
 
-Clicking on :guilabel:`Close` updates the mandate’s end day to the current day. This means that
-invoices issued after the present day will not be processed with an SDD payment.
+To make |sdd| available as an online payment method, :doc:`configure and enable the SEPA Direct
+Debit payment provider <../../payment_providers/sdd>`. Once configured and enabled, it becomes
+available during :doc:`checkout </applications/websites/ecommerce/checkout>` in **eCommerce** and on
+the customer portal when a payment link is used. Customers selecting this payment method are
+prompted to enter their IBAN to complete the :ref:`SEPA Direct Debit mandate
+<accounting/batch_sdd/sdd_mandates>`.
 
-Clicking on :guilabel:`Revoke` disables the mandate immediately. No SDD payment can be registered
-anymore, regardless of the invoice’s date. However, payments that have already been registered are
-still included in the next SDD `.xml` file.
+The |sdd| mandate is then automatically created in :guilabel:`Draft` based on the provided IBAN. To
+validate the information, customers must confirm each mandate with a successful bank transfer of the
+expected amount **using the specified payment reference (communication)**. Once this initial payment
+is received and reconciled, the mandate is automatically validated and updated to the
+:guilabel:`Active` status.
```

Comment:
Shouldn't this part be moved to the `finance/payment_providers/sdd` page? Because the exact payment flow is specific to online payments, and this current page could simply use a "to enable SEPA mandates as an online payment method, go to blablabla".